### PR TITLE
Rename `write_only_fields` to `hidden_fields`

### PR DESCRIPTION
**Pull request by jerabekjiri**
Issue: [AAH-1912](https://issues.redhat.com/browse/AAH-1912)
API PR: https://github.com/ansible/galaxy_ng/pull/1490

After renaming `write_only_fields` to `hidden_fields` and moving them to https://github.com/pulp/pulpcore/pull/3167.  API and UI have to be updated to use `hidden_fields`.  

Pulpcore issue: https://github.com/pulp/pulpcore/issues/2825

I added small tests to verify that the `hidden_fields` are working as expected.

### CODE REVIEW

**Comment by jerabekjiri** [comment]
@himdel FYI I removed `hidden_fields` from the request data, I don't think we need to send them to API. https://github.com/ansible/ansible-hub-ui/pull/2710/files#diff-049213a32cbacbeec291dbc3e5e6809c847a354420364508110b50235ea0103fR170 and https://github.com/ansible/ansible-hub-ui/pull/2710/files#diff-d7ccf0af7fdb44f2f4106924dd93a79d428f3d0289160b2e219486c3a3653d46R18

File: test/cypress/e2e/repo/remote_registry.js [hunk 1]
```diff
@@ -125,7 +125,86 @@ describe('Remote Registry Tests', () => {
...
+    cy.get('.pf-c-expandable-section__toggle-text').click();
+    cy.get(
+      '[data-cy="proxy_username"] > .pf-c-form__group-control > .pf-c-input-group > input',
+    ).should('be.disabled');
+    cy.get(
+      '[data-cy="proxy_password"] > .pf-c-form__group-control > .pf-c-input-group > input',
+    ).should('be.disabled');
+  });
+
+  it.skip('admin can delete data', () => {
```
Comment (himdel) [inline test/cypress/e2e/repo/remote_registry.js:207]: why the `skip`?

Comment (jerabekjiri) [reply test/cypress/e2e/repo/remote_registry.js]: Removed, thanks :+1:, forgot to do that :)

**Comment by jerabekjiri** [comment]
Converting to draft for now. This will have to include new pulp endpoints with `hidden_fields` after updating to the newer pulpcore version. No additional API changes/refactoring will be needed.

**Comment by himdel** [comment]
Closing, EE endpoints API still use write_only_fields, so we're supporting both for now.

**Pull request closed without merging**